tests/python/unittest/test_meta_schedule_tune_relay.py

```diff
@@ -323,6 +326,227 @@ def get_output(data, lib):
         assert np.allclose(actual_output, expected_output, rtol=1e-4, atol=2e-4)
 
 
+@register
+def int32x16(imm, span):
+    return imm.astype("int32x16", span)
+
+
+@T.prim_func
```

Comment:
maybe we could move all the tensorization intrinsics in the same place. Right now for wmma/mma cuda it is `meta_schedule.testing.tir_tensor_intrin` but I believe it is not upstreamed yet.

Reply:
yeah we should do that.

Reply:
Are these generally useful intrinsics or just needed for the tests in this file? If it's just test intrinsics it makes sense to keep here.

Reply:
For now they are only needed here, but for future effort like auto-tensorization, we definitely need a centralized place to put intrinsics so that they can be used from anywhere.